force-app/components/amplitude/lwc/amplitudeAgOversiktAccountDetaljer/__tests__/amplitudeAgOversiktAccountDetaljer.test.js

```diff
@@ -0,0 +1,25 @@
+import { createElement } from 'lwc';
+import AmplitudeAgOversiktAccountDetaljer from 'c/amplitudeAgOversiktAccountDetaljer';
+
+describe('c-amplitude-ag-oversikt-account-detaljer', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('TODO: test case generated by CLI command, please fill in test logic', () => {
```

Comment:
Skal TODO'en med tilhørende kommentar være med? '_TODO: test case generated by CLI command, please fill in test logic'_